### PR TITLE
[Streaming] Lower the minimum streaming rpc interval to 10s

### PR DESCRIPTION
**Pull request by linzhou-db**
Lower the streaming rpc interval to 10s, default is still 30s

### CODE REVIEW

File: client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala [hunk 1]
```diff
@@ -59,7 +59,7 @@ object ConfUtils {
 
   val QUERY_TABLE_VERSION_INTERVAL_SECONDS =
     "spark.delta.sharing.streaming.queryTableVersionIntervalSeconds"
-  val QUERY_TABLE_VERSION_INTERVAL_SECONDS_DEFAULT = "30s"
+  val QUERY_TABLE_VERSION_INTERVAL_SECONDS_DEFAULT = "10s"
```
Comment (chakankardb) [inline client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala:62]: Question: Are we comfortable allowing this for everyone from scalability perspective? Or should we limit it to specific customers first?

Comment (linzhou-db) [reply client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala:62]: discussed offline, the server should be able to handle this.

Comment (linzhou-db) [reply client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala]: reverted

File: client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala [hunk 2]
```diff
@@ -171,8 +171,8 @@ object ConfUtils {
   private def toTimeInSeconds(timeStr: String, conf: String): Int = {
     val timeInSeconds = JavaUtils.timeStringAs(timeStr, TimeUnit.SECONDS)
     validateNonNeg(timeInSeconds, conf)
-    if (conf == QUERY_TABLE_VERSION_INTERVAL_SECONDS && timeInSeconds < 30) {
-      throw new IllegalArgumentException(conf + " must not be less than 30 seconds.")
+    if (conf == QUERY_TABLE_VERSION_INTERVAL_SECONDS && timeInSeconds < 10) {
+      throw new IllegalArgumentException(conf + " must not be less than 10 seconds.")
```
Comment (chakankardb) [inline client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala:175]: Can we define a param for 10 ?

**Review by chakankardb — APPROVED** [review]

File: client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala (hunk 2 above)
Comment (linzhou-db) [reply client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala]: added

**Pull request merged**